Mathlib/Algebra/Module/Submodule/Map.lean

```diff
@@ -0,0 +1,466 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro, Kevin Buzzard, Yury Kudryashov, Frédéric Dupuis,
+  Heather Macbeth
+-/
+
+import Mathlib.Algebra.Module.Submodule.Lattice
+
+/-!
+# `map` and `comap` for `Submodule`s
```

Comment:
Could you add a brief overview of this file to the documentation? I nominate mentioning: `map`, `comap`, `giMapComap` and `gciMapComap`.